data_steward/spec/pages/file_transfer_procedures.md

```diff
@@ -1,43 +1,98 @@
+
+---
+
+## DE-PROVISIONING
+__Step 1:__  Request account to be de-provisioned by sending an email to sysadmin@pmi-ops.org with the subject line “Account De-Provision”.  Include the following information:
+  * Name of Organization or Individual to be de-provisioned.
+  * If it is an individual include a replacement contact.
+
+---
+
+## UPLOADING EHR DATA TO THE DRC
+This part of the process steps through the procedure for uploading EHR data files to the Data and Research Center (DRC) Google Cloud Storage Buckets.The process applies to all program partner organizations that are responsible for sharing participant data (e.g., EHR) with the DRC.  It details the required documentation and parameters needed as well as the testing required to upload EHR data to the DRC Google Cloud Storage Buckets.
+
+__Step 1:__ Log out of any currently active Google IDs. Log into MyAccount (https://myaccount.google.com/)  with the pmi-ops.org credentials provided by the DRC User Admin.
+
+__Step 2:__ Data Stewards will receive an email from the DRC User Admin containing a direct link to the assigned Bucket. Open a web browser and navigate to the Google Cloud Storage Bucket via the direct link shared with you by the DRC User Admin. 
+
+NOTE: If you receive the error message shown below, please check that the pmi-ops.org account is the active account. 
+
+__PICTURE FROM SOP NEEDS INSERTED HERE__
+
+__Step 3:__ Create a folder within the Bucket. Data files for each upload cycle must be contained within a unique folder. Loose files uploaded to the bucket root will not be accepted. 
+
+__PICTURE FROM SOP NEEDS INSERTED HERE__
+
+__Step 4:__ Uploading with gsutil
+- Authenticate:Enter gcloud auth login
+  - This opens a browser window to authenticate to Google Cloud Platform. Login using the assigned pmi-ops account (e.g. john.smith@pmi-ops.org)
+ - Set project: gcloud config set project all-of-us-staging (for example)
+ - Upload files: gsutil cp -r my-site-ehr-data-2017-05-20 gs://aou-ehr-test-awardee-site-123/
+ - Upload
+
+__PICTURE FROM SOP NEEDS INSERTED HERE__
+
+NOTE: for instructions on installing gsutil [click here](https://cloud.google.com/storage/docs/gsutil_install#sdk-install).
+
+_The following is an alternative Upload Procedure to using gsutil_
+ - In a desktop environment, navigate to the data set. Drag and drop the data set files from the local folder into the browser window where the Google Cloud Storage Browser is open to the folder created in Step 3 under Procedures.
+
+__Step 5:__ Within six (6) hours of uploading data to a folder within the Bucket, the [report page](https://storage.googleapis.com/test-aou-curation-spec-site-bucket/report.html) on the All of Us Upload website will display the sites uploaded tables with feedback on whether it was received or loaded. Review this status to verify that the upload was successful.  This website is also provided by the sysadmin in the setup of the pmi-ops account.
```

Comment:
@cmcphilamy this will most likely need to be changed to a production URL, this link will be unstable (it's a test environment)

Reply:
Training using this site will begin this month.  The date will be chosen this week.